cpp/src/platform/windows/LogImpl.cpp

```diff
@@ -41,7 +41,7 @@ errno_t fopen_s(FILE** pFile, const char *filename, const char *mode)
 {
     if (!pFile)
     {
-        _set_errno(EINVAL);
```

Comment:
The Mingw updates were contributed by another user. So its what he used. :) Feel free to open another PR if you don't mind fixing that up.

Thanks